jobs/headforward-python.html

```diff
@@ -0,0 +1,42 @@
+---
+title: Senior Python Developer 
+company: Headforwards Solutions Ltd
+url: https://www.headforwards.com/careers/
+location: Cornwall, United Kingdom
+contract: permanent
+contact:
+name: Jeni Henthorn
+email: jeni@headforwards.com
```

Comment:
The next problem is that `name`, `email` and `phone` properties all need to be indented underneath `contact:`